src/rtc/h5_prep.py

```diff
@@ -509,7 +513,7 @@ def get_metadata_dict(product_id: str,
             url_rtc_algorithm_document = 'https://ieeexplore.ieee.org/document/5752845'
         else:
             raise NotImplementedError
-    metadata_dict['RTC/metadata/processingInformation/algorithms/radiometricTerrainCorrectionAlgorithmReference'] =\
+    metadata_dict['metadata/processingInformation/algorithms/radiometricTerrainCorrectionAlgorithmReference'] =\
         ['rtc_algorithm_reference',
          url_rtc_algorithm_document,
          'A reference to the RTC algorithm applied']
```

Comment:
This is a suggestion. What do you think about adding information about what mosaic mode was used when `is_mosaic` is `True`?

Reply:
Thank you, @seongsujeong . This should be handled in a follow up PR . I'm separating the specs for the burst and mosaic products.